dbtest/dbserver.go

```diff
@@ -66,11 +76,18 @@ func (dbs *DBServer) start() {
 		"--dbpath", dbs.dbpath,
 		"--bind_ip", "127.0.0.1",
 		"--port", strconv.Itoa(addr.Port),
-		"--nssize", "1",
-		"--noprealloc",
-		"--smallfiles",
-		"--nojournal",
+		"--storageEngine=" + dbs.engine,
 	}
+	if dbs.engine == "mmapv1" {
+		// default to mmapv1 and add mmapv1-only args (nssize, noprealloc, smallfiles and nojournal)
+		dbs.args = append(dbs.args,
```

Comment:
This should be ```args``` instead of ```dbs.args``` (I'm surprised there was no error since dbs object doesn't have args anymore).